bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -214,9 +217,11 @@ public BatchExecutor(
    */
   ListenableFuture<com.google.bigtable.v1.Row> issueGetRequest(Get get) {
     LOG.trace("issueGetRequest(Get)");
-    ReadRowsRequest.Builder builder = getAdapter.adapt(get);
```

Comment:
Is there any reason that getAdapter.adapt(get) doesn't create a DefaultReadHooks if a parameter isn't supplied?